app/views/events/index.html.erb

```diff
@@ -6,14 +6,18 @@
 
 <div class="row">
   <div class="event-list col-md-10 col-md-offset-1">
-    <% if can? :new, Event %>
     <div class="clearfix">
       <%= link_to new_event_path, :class => 'btn btn-primary pull-right' do
         glyph('plus') + ' ' +
         t('.new', :default => t('helpers.links.new'))
+      end if can? :new, Event %>
```

Comment:
this is actually a "suffix" if. believe me if I tell you that it was even more confusing the other way around